Recommend not to run the game on a hybrid graphics setup in `Performance troubleshooting` (#8260)
ppy/osu-wiki: wiki/Performance_troubleshooting/en.md

- Is your frame limiter set to `Unlimited`? Set it to `Optimal` and see if it helps. See [its section below](#unlimited) for more information on why `Unlimited` is usually not the best choice.
- Are you running windowed? Switch to fullscreen.
- Have you updated your drivers (especially GPU)? Try doing so and see if things improve. Alternatively, if you recently updated them, try rolling back.

> - Are you using a hybrid graphics setup (such as NVIDIA Optimus)? If that is the case, try running the game on only one of them. For running the game only on the dedicated GPU, you can use a MUX switch if available, or you can connect the computer to an external display.

Reviewer: ```suggestion
- Are you using a hybrid graphics setup such as NVIDIA Optimus? If that is the case, try running the game on only the dedicated GPU, by using a MUX switch if available, or connecting the computer to an external display.
```

Author: Many people don't need to run osu! on the dedicated GPU. osu! is capable of running on the integrated GPU, especially for modern iGPU. I can run like more than 500 FPS with my iGPU, which is more than enough.

> - Are you using a hybrid graphics setup (where the dedicated GPU renders the game and the integrated GPU displays the screen)? If so, try running the game on either only integrated GPU or only dedicated GPU (by using a mux switch if available, or by using an external display).

Reviewer 2: I dunno about this advice. The only case where this causes issues is if the user is running stupid high frame rates. Also, to make this understandable I'd just call it out for what it is – refer to it using the "optimus" branding because the only time this even comes up is with optimus chips. Then you don't need to try and explain to the user the rendering pipeline.

> - Are you using a hybrid graphics setup (where the dedicated GPU renders the game and the integrated GPU displays the screen)? If so, try running the game on either only integrated GPU or only dedicated GPU (by using a mux switch if available, or by using an external display).

Reviewer 3: Then at least, I think this article should mention something about optimus, and leave it there for people to search about it themselves, rather than having no info.
I'd also like to have a part about this on official wiki page, since most of the time someone ask for tech support, I literally have to search for some of your tweets (luckily I bookmarked them) just to tell them about this problem.

> - Are you using a hybrid graphics setup (where the dedicated GPU renders the game and the integrated GPU displays the screen)? If so, try running the game on either only integrated GPU or only dedicated GPU (by using a mux switch if available, or by using an external display).

Reviewer 2: Again, I'm not against that. I'm saying to use the "optimus" term instead of "hybrid graphics setup (where the dedicated GPU renders the game ...)". A very small subset of users will understand the second. Most will choose to not read the sentence.

> - Are you using a hybrid graphics setup (where the dedicated GPU renders the game and the integrated GPU displays the screen)? If so, try running the game on either only integrated GPU or only dedicated GPU (by using a mux switch if available, or by using an external display).

Reviewer 4: a passerby comment: if this is still an issue, you can refer to "Optimus" in the body of the article and add a footnote that would mention that it is also valid for other hybrid setups (and explain what they are)

> - Are you using NVIDIA Optimus or any other hybrid graphics setup? If that is the case, try running the game on only one of them. Your integrated GPU might already be capable of running osu! at a sufficient frame rate. For running the game only on the dedicated GPU, you can use a MUX switch if available, or you can connect the computer to an external display.

Reviewer 2: ```suggestion
- Does your PC use an NVIDIA Optimus or similar hybrid graphics setup? If so, try forcing the game to run on each of the GPUs to compare performance. It is quite common for the integrated GPU to provide more consistent performance for games like osu!. Search for "Graphics settings" in windows and set osu! to either "Power Saving" or "High Performance" to experiment.
```

How about this?
